addons/nuxeo-retention/nuxeo-retention-core/src/main/java/org/nuxeo/retention/operations/AttachRetentionRule.java

```diff
@@ -34,7 +34,7 @@
 /**
  * @since 11.1
```

Comment:
Not sure about the guidelines for this kind of stuff.
We are backporting something that was introduced for `11.1`. Should we keep the `@since 11.1`? @troger

Reply:
agree we can keep it as it is :)